sentry-log4j2/src/main/java/io/sentry/log4j2/SentryAppender.java

```diff
@@ -187,6 +187,7 @@ public void append(final @NotNull LogEvent eventObject) {
         CollectionUtils.filterMapEntries(
             loggingEvent.getContextData().toMap(), entry -> entry.getValue() != null);
     if (!contextData.isEmpty()) {
+      List<String> contextTags = HubAdapter.getInstance().getOptions().getContextTags();
```

Comment:
```suggestion
      List<String> contextTags = hub.getOptions().getContextTags();
```
we actually have a `hub` instance already as a class member, can we use that one instead?